hack/save-logs-files.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Copyright 2023 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+# if a uid is passed, another file will be created, containing only logs with the uid
+fybrik_application_uid=$1
```

Comment:
please add the command line parsing, such has https://www.baeldung.com/linux/bash-parse-command-line-arguments, and add the "help" (Usage) option

Reply:
done